mailpoet/tests/_support/IntegrationTester.php

```diff
@@ -98,4 +98,18 @@ public function deleteTestWooOrders() {
   public function uniqueId($length = 10): string {
     return Security::generateRandomString($length);
   }
+
+  /**
+   * Compares two DateTimeInterface objects by comparing timestamp values.
+   * $delta parameter specifies tolerated difference
+   */
+  public function assertEqualDateTimes(DateTimeInterface $date1 = null, DateTimeInterface $date2 = null, int $delta = 0) {
```

Comment:
Can I ask why you use all parameters as optional with default values?
In my opinion, the first two parameters can be nullable without a default value. Because if I use this function without `date2`, it throws an exception. Or am I missing something?

I like this idea 👍

Reply:
@lysyjan Good catch! That's exactly what I wanted to do, but my hands were probably faster than my head... Updated!